examples/tutorials/analysis-time/Variability_estimation.py

```diff
@@ -0,0 +1,266 @@
+    )
+)
+
+sig_A_mp = A_mp_100 / A_mp_err
+
+print(sig_A_mp)
+
+######################################################################
+#  Fractional excess variance, point-to-point fractional variance and doubling/halving time
+# ----------------------------------------------
+# The fractional excess variance ([reference
+# paper](https://ui.adsabs.harvard.edu/abs/2003MNRAS.345.1271V/abstract)) is a simple but effective method to assess the significance of a time variability feature in an object, for example an AGN flare. It is important to note that it requires gaussian errors to be applicable. The excess variance computation is implemented in the `estimators.utils` subpackage of `gammapy`. A similar estimator is the point-to-point fractional variance, which samples the lightcurve with a smaller time granularity. In general, the point-to-point fractional variance being higher than the fractional excess variance is a sign of the presence of very short timescale variability. The point-to-point variability is also implemented in the `estimators.utils` subpackage of `gammapy`.
+#
+# In the same subpackage `gammapy` also offers the computation of the doubling and halving time of the lightcurve, an estimator which gives information on the shape of the variability feature.
+
+from gammapy.estimators.utils import (
+    compute_lightcurve_doublingtime,
+    compute_lightcurve_fpp,
+    compute_lightcurve_fvar,
+)
+
+fvar_table = compute_lightcurve_fvar(lc_1d)
+print(fvar_table)
+
+fpp_table = compute_lightcurve_fpp(lc_1d)
+print(fpp_table)
+
+dtime_table = compute_lightcurve_doublingtime(lc_1d, flux_quantity="flux")
+print(dtime_table)
+
+######################################################################
+# ## Bayesian blocks
+# ----------------------------------------------
+# The presence of temporal variability in a lightcurve can be assessed by using bayesian blocks ([reference
+# paper](https://ui.adsabs.harvard.edu/abs/2013ApJ...764..167S/abstract)). A good and simple-to-use implementation of the algorithm is found in `astropy.stats`([documentation](https://docs.astropy.org/en/stable/api/astropy.stats.bayesian_blocks.html)). This implementation uses guassian statistics, as opposed to the [first introductory paper](https://iopscience.iop.org/article/10.1086/306064) which was based on poissonian statistics.
+#
+# By passing the flux and error on the flux as `measures` to the method we can obtain the list of optimal bin edges defined by the bayesian blocks algorithm.
+
+time = lc_1d.geom.axes["time"].time_mid.mjd
+
+bayesian_edges = Time(
+    bayesian_blocks(
+        t=time, x=flux.flatten(), sigma=flux_err.flatten(), fitness="measures"
+    ),
+    format="mjd",
+)
+
+# We then can then (optionally) compute the new lightcurve and plot it
+
+time_intervalsB = [
+    Time([tstart, tstop])
+    for tstart, tstop in zip(bayesian_edges[:-1], bayesian_edges[1:])
+]
+
+lc_maker_1d = LightCurveEstimator(
+    energy_edges=[0.3, 10] * u.TeV,
```

Comment:
why change the `energy_edges` from the original light curve?
It can be interesting to plot `lc_1d` and `lc_b` on the same axis